client/webserver/site/src/html/forms.tmpl

```diff
@@ -485,7 +485,8 @@
     <span class="fs16 grey">%</span>
   </div>
   <div class="mt-1 flex-center flex-row fs14 d-hide" data-tmpl="syncRemainBox">
-    <span data-tmpl="syncRemain" class="me-1"></span> [[[remaining]]]
+    <span data-tmpl="syncRemaining"><span data-tmpl="syncRemain" class="me-1"></span> [[[remaining]]]</span>
+    <span data-tmpl="syncFinishingUp" class="me-1"></span>
```

Comment:
Reasons not to do `[[[finishing_up]]]` and instead setting `syncFinishingUp.textContent` in JS?  There's nothing dynamic about it and we show/hide the entire `span`?

Reply:
> we show/hide the entire span?

yes, we hide one of **syncRemaining**/**syncFinishingUp** and show the other,

yeah, seems like could be easily moved to go html templates, done (and re-tested, working same way).